examples/demo/scene/isocurve_for_trisurface_qt.py

```diff
@@ -96,7 +96,7 @@ def update_view(self):
 class Canvas(scene.SceneCanvas):
 
     def __init__(self):
-        scene.SceneCanvas.__init__(self, keys='interactive')
+        scene.SceneCanvas.__init__(self, keys=None)
```

Comment:
This is an example of embedding the canvas within a Qt window, so the interactive keys like "escape" and "fullscreen" aren't useful. Hitting escape just makes the plot go away and the window stays open with controls, etc.